modules/apps/transfer/internal/events/events.go

```diff
@@ -11,16 +11,18 @@ import (
 )
 
 // EmitTransferEvent emits a ibc transfer event on successful transfers.
-func EmitTransferEvent(ctx sdk.Context, sender, receiver string, tokens types.Tokens, memo string) {
-	jsonTokens := mustMarshalType[types.Tokens](tokens)
+func EmitTransferEvent(ctx sdk.Context, sender, receiver string, tokens types.Tokens, memo string, forwarding types.Forwarding) {
+	tokensBz := mustMarshalType[types.Tokens](tokens)
+	forwardingBz := mustMarshalType[types.Forwarding](forwarding)
```

Comment:
Since `mustMarshalType` returns a `string`, should these variables be `tokensStr` and `forwardingStr` instead?